### PR TITLE
Update motivation

### PR DESCRIPTION
**Pull request by Jack-Works**
cc @ljharb

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
-# await.ops proposal
+# await operation proposal
```
Comment (ljharb) [inline README.md]:
```suggestion
# await operations proposal
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+Introduce await.all / await.race / await.allSettled / await.any to simplify the usage of async functions
 
 Stage: 1
 
 Champions: [Jack Works](https://github.com/Jack-Works), [Jordan Harband](https://github.com/ljharb)
 
 ## Motivation
 
-To simplify the usage of a set of Promises. (Previous discussions: https://es.discourse.group/t/allow-awaiting-on-arrays/178/19)
+When developers use async functions, they have to know about Promise if they want to handle multiple tasks parallelly (`Promise.all` stuff), this is some kind of "abstraction leak".
```
Comment (ljharb) [inline README.md]:
```suggestion
When developers use async functions, they have to know about Promise if they want to handle multiple tasks concurrently (`Promise.all`, eg), this is some kind of "abstraction leak".
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
+
+This proposal is intended to fix the problem by introducing parallel Promise utils in the async-await style.
```
Comment (ljharb) [inline README.md]:
```suggestion
This proposal is intended to fix the problem by introducing concurrent Promise utils in the async-await style.
```

File: README.md [hunk 4]
```diff
@@ -37,3 +43,12 @@ await.allSettled expr
...
 ```
+
+## Alternative solution
+
+Introduce a new global `Async` (or `Await`), then let
+
+- `Async.all` (equals to `Promise.all`)
+- `Async.race` (equals to `Promise.race`)
+- `Async.allSettled` (equals to `Promise.allSettled`)
+- `Async.any` (equals to `Promise.any`)
```
Comment (ljharb) [inline README.md]: I’m very confused by this - adding a new global isn’t going to address the confusion caused by the ability to use syntax for await but not for the combinators.

Comment (Jack-Works) [reply README.md]: oh yeah, I found this way we need `await Async.all(...)` which is, ... a bit strange. let me remove this part.

**Comment by Jack-Works** [comment]
I hope this new motivation (other than "save 9 characters") can help to optimize the mental model of using async functions.

**Comment by ljharb** [comment]
Now it just saves 2, though, and it’s not syntax.

**Comment by Jack-Works** [comment]
> Now it just saves 2, though, and it’s not syntax.

I removed that "new global" part

**Review by ljharb — APPROVED** [review]

**Pull request merged**